pom.xml

```diff
@@ -1361,6 +1361,73 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>yarn2-execution</id>
+      <activation>
+        <file>
+          <exists>.mvn_exec_yarn2</exists>
```

Comment:
I guess the same applied for yarn3?  (the --mutex flag is a yarn 1 thing right?)